src/main/webapp/styles/main.css

```diff
@@ -139,11 +172,188 @@ img {
 
 #header-logo {
     float: left;
-    width:150px;
+    width: 150px;
     display: inline;
     margin-top: 25px;
 }
 
 .highlight-span {
     color: var(--primaryDark);
+}
+
+
+#appointments {
+    font-family: Arial, Helvetica, sans-serif;
+    border-collapse: collapse;
+    width: 100%;
+}
+
+#appointments td, #appointments th {
+    border: 1px solid #ddd;
+    padding: 8px;
+    text-align: center;
+}
+
+#appointments th {
+    padding-top: 12px;
+    padding-bottom: 12px;
+    text-align: left;
+    background-color: #e6f3ff;
+    color: black;
+    text-align: center;
+    position: sticky;
+    top: 0;
+    z-index: 2
+}
+#slots {
+    text-align: center;
+    width: 50%;
+    padding: 10px;
+    font-size: 20px;
+    border-radius: 2px;
+}
+
+#options {
+    border-radius: 45%;
+    width: 110px;
+    height: 100px;
+    outline: 0;
+    font-size: 15px;
+    background-color: #e6f3ff;
+}
+
+.blueAlternateButton {
```

Comment:
My bad. These classes are unused. Could we please delete blueAlternateButton and pinkAlternateButton.

Reply:
Hmm, I see you have made some use of them actually.